### DISCUSSION TITLE
# shellcheck disable as a hyperlink

### DISCUSSION THREAD

**Discussion by levlist** [timonwong/vscode-shellcheck#299] [Ideas]
Hi,

when I review other developer's code and I often run such cases when they ignore a given directive in their code.
For example:
`# shellcheck disable=SC2116,SC2086`

It could be a helpful feature if vscode-shellcheck could transform these directive IDs to clickable hyperlinks, which would redirect the user to the description of the SC ID.

Example of my idea:
> #shellcheck disable=[SC2116](https://github.com/koalaman/shellcheck/wiki/SC2116),[SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)

What is your opinion about it?

Br,
Viktor

**Comment by felipecrs** [comment]
This is such an awesome feature request! I agree 100%.

@timonwong this might be a reason why we could introduce https://github.com/vscode-shellcheck/vscode-shellcheck/issues/114.

**Comment by timonwong** [comment]
Landed in #539
